docs/en/tutorials/getting-started.rst

```diff
@@ -157,12 +157,12 @@ The second block consists of the instantiation of the ORM
 ``Configuration`` object using the Setup helper. It assumes a bunch
 of defaults that you don't have to bother about for now. You can
 read up on the configuration details in the
-`reference chapter on configuration <../reference/configuration.rst>`_.
+`reference chapter on configuration <http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/configuration.html>`_.
```

Comment:
RST doesn't just compile to HTML ;-)
On Jan 2, 2016 04:17, "Hari K T" <notifications@github.com> wrote:

> In docs/en/tutorials/getting-started.rst
> <https://github.com/doctrine/doctrine2/pull/5583#discussion_r48682128>:
>
> > @@ -157,12 +157,12 @@ The second block consists of the instantiation of the ORM
> >  ``Configuration`` object using the Setup helper. It assumes a bunch
> >  of defaults that you don't have to bother about for now. You can
> >  read up on the configuration details in the
> > -`reference chapter on configuration <../reference/configuration.rst>`_.
> > +`reference chapter on configuration <http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/configuration.html>`_.
>
> What is the point of internal link in a website ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/doctrine/doctrine2/pull/5583/files#r48682128>.
>